https://lists.mindrot.org/pipermail/openssh-unix-dev/2016-December/035604.html

From dca2985bff146f756b0019b17f08c35f28841a04 Mon Sep 17 00:00:00 2001
From: Mike Frysinger <vapier@gentoo.org>
Date: Mon, 19 Dec 2016 15:59:00 -0500
Subject: [PATCH] regress/allow-deny-users.sh: fix bashism in test

The test command uses = for string compares, not ==.  Using some POSIX
shells will reject this statement with an error about an unknown operator.
---
 regress/allow-deny-users.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/regress/allow-deny-users.sh b/regress/allow-deny-users.sh
index 32a269afa97c..86805e19322b 100644
--- a/regress/allow-deny-users.sh
+++ b/regress/allow-deny-users.sh
@@ -4,7 +4,7 @@
 tid="AllowUsers/DenyUsers"
 
 me="$LOGNAME"
-if [ "x$me" == "x" ]; then
+if [ "x$me" = "x" ]; then
 	me=`whoami`
 fi
 other="nobody"
-- 
2.11.0.rc2

